Formula/ooniprobe.rb

```diff
@@ -1,9 +1,9 @@
 class Ooniprobe < Formula
   desc "Network interference detection tool"
   homepage "https://ooni.org/"
-  url "https://github.com/ooni/probe-cli/archive/v3.10.1.tar.gz"
-  sha256 "2b81c14133f39ac91c4ea6761be7a27d768cd88989b52ae72376d1d7b69de322"
-  license "BSD-3-Clause"
+  url "https://github.com/ooni/probe-cli/archive/v3.13.0.tar.gz"
+  sha256 "a055aed8c2d0d898b7cdb843cf247cf3b593c8ac7045103c08b3088b7d4d1737"
+  license "GPL-3.0"
```

Comment:
confirmed: https://github.com/ooni/probe-cli/pull/641

```suggestion
  license "GPL-3.0-or-later"
```

Reply:
Nice!